spectec/src/el/free.ml

```diff
@@ -60,18 +66,21 @@ and free_typ t =
 and free_typfield (_, t, _) = free_typ t
 and free_typcase (_, ts, _) = free_list free_typ ts
 
-
 (* Expressions *)
 
 and free_exp e =
   match e.it with
   | VarE id -> free_varid id
   | AtomE _ | BoolE _ | NatE _ | TextE _ | EpsE | HoleE _ -> empty
-  | UnE (_, e1) | DotE (e1, _) | LenE e1
-  | ParenE (e1, _) | BrackE (_, e1) -> free_exp e1
-  | BinE (e1, _, e2) | CmpE (e1, _, e2)
-  | IdxE (e1, e2) | CommaE (e1, e2) | CompE (e1, e2)
-  | InfixE (e1, _, e2) | FuseE (e1, e2) ->
+  | UnE (_, e1) | DotE (e1, _) | LenE e1 | ParenE (e1, _) | BrackE (_, e1) ->
+    free_exp e1
+  | BinE (e1, _, e2)
+  | CmpE (e1, _, e2)
+  | IdxE (e1, e2)
+  | CommaE (e1, e2)
+  | CompE (e1, e2)
+  | InfixE (e1, _, e2)
+  | FuseE (e1, e2) ->
```

Comment:
This is already done by 5ec0fe6d, though it doesn't work in this particular case since the line is 130 characters wide. We can increase the margin if we want.